xla/debug_options_flags.cc

```diff
@@ -1783,6 +1784,16 @@ void MakeDebugOptionsFlags(std::vector<tsl::Flag>* flag_list,
       "Einsums that have partitioned operand(can be either LHS or RHS) that's "
       "larger than this threshold will be transformed to use windowed einsums."
       "Default is 100000"));
+  flag_list->push_back(tsl::Flag(
+      "xla_gpu_total_bytes_threshold_for_windowed_einsum",
+      int64_setter_for(
+          &DebugOptions::set_xla_gpu_total_bytes_threshold_for_windowed_einsum),
+      debug_options->xla_gpu_total_bytes_threshold_for_windowed_einsum(),
+      "This controls whether to enable windowed einsum (collective matmul) "
+      "based on the total flops(combined sizes of both operands) if set >= 0."
```

Comment:
Remove the word flops here. The line should be

> based on the sum of sizes of the two operands if set >= 0.

Also remove the word "flops" from all other parts of the PR, since it is not based on flops.